ddd/admission/domain/service/i_calendrier_inscription.py

```diff
@@ -213,7 +213,7 @@ def verifier_reorientation_renseignee_si_eligible(
             and (
                 proposition.est_reorientation_inscription_externe is None
                 or proposition.est_reorientation_inscription_externe
-                and not proposition.attestation_inscription_reguliere
+                and (not proposition.attestation_inscription_reguliere or not proposition.formulaire_reorientation)
```

Comment:
A voir ce que t'en dis mais je préfère un not (A and B) que (not A or not B)

Qu'en penses-tu ?

Reply:
C'est peut-être plus simple à lire effectivement, je vais modifier.